grails-app/services/org/broadinstitute/orsp/DataUseLetterService.groovy

```diff
@@ -42,9 +43,9 @@ class DataUseLetterService {
         }
     }
 
-    DataUseRestriction createSdul(DataUseRestriction restriction, String displayName) {
+    DataUseRestriction saveRestriction(DataUseRestriction restriction, String displayName) {
         Issue consent = queryService.findByKey(restriction.consentGroupKey)
-        def updatedOrCreated = "Created"
+        def updatedOrCreated = "Updated"
```

Comment:
This is previous code that I missed, sorry. Avoid `def` when possible. Also, this could be a lot simpler:
```
String updatedOrCreated = (restriction.id == null) ? "Created" : "Updated"
```